ui/nim-status-client.pro

```diff
@@ -23,6 +23,7 @@ TRANSLATIONS += \
 
 OTHER_FILES += $$files("$$PWD/*.qml", true)
 OTHER_FILES += $$files("$$PWD/*.js", true)
+OTHER_FILES += $$files("$$PWD/../src/*.nim", true)
```

Comment:
is it intentional? If we want to add it to the config it should go to a separate pr imo :)